Gemfile.lock

```diff
@@ -375,23 +381,24 @@ GEM
     thread_safe (0.3.5)
     tilt (2.0.5)
     ttfunk (1.4.0)
-    turbolinks (2.5.3)
-      coffee-rails
+    turbolinks (5.0.1)
```

Comment:
Looks like they've added TL5 support to `wice_grid` now 😄
I would still like to test this on `stage` before merging, considering the problems we had on `voting`.